docs/usage/quickstart.rst

```diff
@@ -217,23 +228,36 @@ Similarly, we can list all deployed Helm_ applications::
-``kubectl proxy``. Then, while the tunnel is up and running, the following
-tools are available:
-
-+-------------------------+---------------------------------------------------------+-------------------------------------------------------------------------------------------------+
-| Service                 | Role                                                    | Link                                                                                            |
-+=========================+=========================================================+=================================================================================================+
-| `Kubernetes dashboard`_ | A general purpose, web-based UI for Kubernetes clusters | http://localhost:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/ |
-+-------------------------+---------------------------------------------------------+-------------------------------------------------------------------------------------------------+
-| `Grafana`_              | Monitoring dashboards for cluster services              | http://localhost:8001/api/v1/namespaces/kube-ops/services/kube-prometheus-grafana:http/proxy/   |
-+-------------------------+---------------------------------------------------------+-------------------------------------------------------------------------------------------------+
-| `Cerebro`_              | An administration and monitoring console for            | http://localhost:8001/api/v1/namespaces/kube-ops/services/cerebro:http/proxy/                   |
-|                         | Elasticsearch clusters                                  |                                                                                                 |
-+-------------------------+---------------------------------------------------------+-------------------------------------------------------------------------------------------------+
-| `Kibana`_               | A search console for logs indexed in Elasticsearch      | http://localhost:8001/api/v1/namespaces/kube-ops/services/http:kibana:/proxy/                   |
-+-------------------------+---------------------------------------------------------+-------------------------------------------------------------------------------------------------+
+Services to operate and monitor your MetalK8s cluster are provided. To
+access these dashboards:
+
+1. The credentials in :file:`inventory/quickstart-cluster/artifacts/admin.conf`
+   must be present. If you are accessing the cluster using the same machine
+   from which you installed MetalK8s, this path is already set in the
+   ``make shell`` virtual environment. If you are accessing the cluster using
+   a machine from which you didn't install MetalK8s, copy the credentials in
+   admin.conf and export its path with::
+
+       $ export KUBECONFIG=[...]/metalk8s/inventory/quickstart-cluster/artifacts/admin.conf
```

Comment:
I'd replace this instance with a more generic path. When copying `admin.conf` to another machine, there's no need to recreate the whole inventory directory structure.